gui/system/alertmods/service_monitor.py

```diff
@@ -0,0 +1,21 @@
+import os
+from freenasUI.system.alert import alertPlugins, Alert, BaseAlert
+
+
+class ServiceMonitor(BaseAlert):
+
+    def run(self):
+
+        alerts = []
+
+        for file in os.listdir("/tmp/"):
+            if file.endswith(".service_monitor"):
+                full_path = '/tmp/' + file
+                with open(full_path, 'r') as _file:
+                    for alert_line in _file:
+                        alerts.append(Alert(Alert.WARN, alert_line))
```

Comment:
Using sentinel files was a hack we did for a lack of better interface, we did that in several places, and I am ashamed of them :P 
How do you feel about providing an interface in the middlewared plugin for checking the service monitor status?